docs/admin/service-accounts-admin.md

```diff
@@ -64,6 +64,11 @@ TokenController runs as part of controller-manager. It acts asynchronously. It:
 - observes secret addition, and ensures the referenced ServiceAccount exists, and adds a token to the secret if needed
 - observes secret deletion and removes a reference from the corresponding ServiceAccount if needed
 
+Note that token controller feature in controller-manager need a service account private key file passed by
```

Comment:
Suggested edit:
You must pass a service account private key file to the token controller in the controller-manager by using the \`--service-account-private-key-file\` option. The private key will be used to sign generated service account tokens. Similarly, you must pass the corresponding public key to the kube-apiserver using the \`--service-account-key-file\` option. The public key will be used to verify the tokens during authentication.